### PR TITLE
Update tutorials and update small Insights style changes

### PR DESCRIPTION
**Pull request by edward-io**
Captum Insights:

- Reduce padding around buttons for notebooks
- Remove matplotlib grid for image features
- Fix newline after every word in text features
- Update screenshots

Notebooks:

- Add link to download models in notebook tutorials

### CODE REVIEW

File: tutorials/CIFAR_TorchVision_Interpret.ipynb [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "In this tutorial we build a simple model as described in:\n",
     "https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py\n",
     "\n",
-    "Then we use attribution algorithms such as `IntegratedGradients`, `Saliency`, `DeepLift` and `NoiseTunnel` to attribute the label of the image to the input pixels and visualize it.\n  ",
-    "\n  ",
-    "**Note:** Before running this tutorial, please install the torchvision, and matplotlib packages."
+    "Then we use attribution algorithms such as `IntegratedGradients`, `Saliency`, `DeepLift` and `NoiseTunnel` to attribute the label of the image to the input pixels and visualize it.\n",
+    "  \n",
+    "  **Note:** Before running this tutorial, please install the torchvision, and matplotlib packages."
```
Comment (NarineK) [inline tutorials/CIFAR_TorchVision_Interpret.ipynb]: Were these intended changes ?

File: tutorials/Titanic_Basic_Interpret.ipynb [hunk 2]
```diff
@@ -835,7 +835,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.7.6"
```
Comment (NarineK) [inline tutorials/Titanic_Basic_Interpret.ipynb]: Looks like python version changed. Here. We should probably agree on one and use that for all notebooks.

File: tutorials/CIFAR_TorchVision_Interpret.ipynb (hunk 1 above)
Comment (edward-io) [reply tutorials/CIFAR_TorchVision_Interpret.ipynb]: It was automatically done by the notebook.

File: tutorials/Titanic_Basic_Interpret.ipynb (hunk 2 above)
Comment (edward-io) [reply tutorials/Titanic_Basic_Interpret.ipynb]: I think the notebook should run the same for every Python version, I just had a different one than @vivekmig 's.

**Pull request closed without merging**